simple-gotracing/capture_args.pxl

```diff
@@ -1,5 +1,8 @@
 # -*- mode: python; -*-
 
+# Copyright (c) Pixie Labs, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License")
+
```

Comment:
Did you mean to copy only part of the licensing text?

Reply:
This is how it was in `./slack-alert-app/go/http_errors.pxl`, so just trying to keep things consistent.